admin-manual/maintenance/maintenance.rst

```diff
@@ -134,72 +142,280 @@ The index names are: `aips`, `aipfiles`, `transfers` and `transferfiles`.
+
+Data to back up from an Archivematica instance:
+
+#. MCP database (see below for details)
+#. SS database (see below for details)
+#. Elasticsearch index (see below for details)
+#. Pointer files (in the Storage Service internal processing location; the
+   default location is /var/archivematica/storage_service)
+#. AM config in /etc/archivematica
+#. defaultProcessingMCP.xml (processing configuration, in 
+   /var/archivematica/sharedDirectory/sharedMicroServiceTasksConfigs/processingMCPConfigs) 
+
+
+If doing an update or migration of Archivematica to a new server, the following
+may also be important to back up:
+
+#. Archivematica source code (/opt/archivematica ) (to know which version of the
+   software was installed, if there were custom changes, etc.)
+#. Archivematica shared directory (/var/archivematica/sharedDirectory/)
+
+If your instance uses automation-tools, that should also be backed up:
+
+#. Source code ( /opt/archivematica/automation-tools)
+#. Scripts (normally in /etc/archivematica/automation-tools)
+#. Crontab entries for automation-tools 
+#. Automation database (normally in /var/archivematica/automation-tools/)
+#. Any other helper scripts source and databases
+
+
+Archivematica Database backup and restore
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
 MySQL is used to store short-term processing data. You can back up the MySQL
 database by using the following command:
 
 .. code:: bash
 
-   mysqldump -u <your username> -p<your password> -c MCP > <filename of backup>
+   mysqldump -u <your username> -p <your password> -c MCP > <filename of backup>
+
+
+To restore from mysqldump file:
+
+.. code:: bash
+
+   mysql –user yourusername -p <your password> MCP < MCP_backup.sql
+
+Storage Service Database backup and restore
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To backup the SQLite database and pointer files created by the storage service run:
+
+.. code:: bash
+
+  rsync -av /var/archivematica/storage_service /backup/location/storage_service
+  rsync -av /var/archivematica/storage-service/storage.db /backup/location/storage.db
+
+Note: The Storage Service must not be actively in use. Make sure the
+Storage Service is not running by stopping the nginx or storage-service
+services or by making the backup at a time that it is not in use.
+
+To restore Storage Service from backup:
+
+.. code:: bash
+
+  service archivematica-storage-service stop
+  rsync -av /backup/location/storage.db /var/archivematica/storage-service/storage.db
+  rsync -av /backup/location/storage_service /var/archivematica/storage_service
+  service archivematica-storage-service start
+
+Elasticsearch
+^^^^^^^^^^^^^
 
 Elasticsearch is used to store long-term data. Instructions and scripts for
 backing up and restoring Elasticsearch are available in the
 `Elasticsearch documentation`_.
 
+**Preconfiguration**
+
+The path.repo and snapshot repository have to be configured. For example, using
+/var/lib/elasticsearch/backup-repo as the repo path:
```

Comment:
I often give paths a bit of code formatting; not required, but it can help with clarity of reading I find. Totally up to you though, there's a lot of code in this section!